docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,121 @@
+# Publish and Share via Azure
+
+## Step 1: Install dependencies
+
+```bash
+pip install 'kedro-viz[azure]'
+```
+
+## Step 2: Configure your Azure Blob Storage
+
+You can host your Kedro-Viz project on Azure Blob Storage. You must first create an Azure Storage account and then enable static website hosting. To do so, follow the [Azure tutorial](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-how-to?tabs=azure-portal) to configure a static website on AzureBlobStorage.
```

Comment:
Can we number the steps inside the step `Configure your Azure Blob Storage` ?

Reply:
Resolved.